.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: Build uOFW
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-16.04
```

Comment:
Ubuntu `18.04` and `20.04` have a more recent GCC that has stricter checks.

We'll need to update the `utils/` before upgrading.